guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>Suggestion: spelling of "labeling" needs to be consistent throughout the document.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated.